src/pages/log-match/logMatch.tsx

```diff
@@ -71,14 +75,70 @@ export const match = new Elysia({
         blackPlayerOne: black1Id,
         blackPlayerTwo: black2Id ? black2Id : null,
         seasonId: activeSeason.id,
+        createdAt: new Date(),
       };
 
-      const insertResult = await writeDb.insert(matches).values(matchInsert);
+      const matchesForQuests = await getMatchesLessThanEqual(
+        activeSeason.id,
+        matchInsert.createdAt,
+      );
+
+      const newMatchWithPlayer: Match = {
```

Comment:
Ikke en stor fan af den her tomme match 😬

Reply:
Har du nogen gode forslag til at undgå det? :D

Reply:
How bout this -> #77